src/main/scala/uk/gov/nationalarchives/tdr/api/auth/ValidationAuthoriser.scala

```diff
@@ -21,14 +21,6 @@ class ValidationAuthoriser(implicit executionContext: ExecutionContext)
                            ctx: Context[ConsignmentApiContext, _]
                           ): BeforeFieldResult[ConsignmentApiContext, Unit] = {
 
-    // All fields must have an authorisation tag defined. This means that if we forget to add authorisation, the
```

Comment:
Probably a stupid idea, but, I'm wondering if we could actually pass in the consignment id, when calling the `getClosureMetadata`? Then check the user has access to the consignment (`ValidateUserHasAccessToConsignment`). I would think the consignment id is going to be available via the URL for the closure metadata form, eg `/consignmentid/fileid/closureMetadataForm`. At least then there would be some check. It would also stop a user from trying to update the custom metadata from another users upload?

@MancunianSam What do you think? I would quite like to keep this constraint.